compiler/rustc_error_codes/src/error_codes/E0010.md

```diff
@@ -5,7 +5,5 @@ the heap at runtime, and therefore cannot be done at compile time.
 Erroneous code example:
 
 ```compile_fail,E0010
```

Comment:
This is now E0015

Reply:
Should this be removed then, or just replaced with `#[rustc_box]`?